README.md

```diff
@@ -196,6 +196,8 @@ The following table lists the configurable parameters of the Harbor chart and th
 | `updateStrategy.type`                                                | The update strategy for deployments with persistent volumes(jobservice, registry and chartmuseum): `RollingUpdate` or `Recreate`. Set it as `Recreate` when `RWM` for volumes isn't supported                                                                                                                                                                                                                                                                     | `RollingUpdate`                       |
 | `logLevel`                                                           | The log level: `debug`, `info`, `warning`, `error` or `fatal`                                                                                                                                                                                                                                                                                                                                                                                                     | `info`                                |
 | `harborAdminPassword`                                                | The initial password of Harbor admin. Change it from portal after launching Harbor                                                                                                                                                                                                                                                                                                                                                                                | `Harbor12345`                         |
+| `existingSecretAdminPassword`                                                | The name of secret whera admin password can be found. Harbor                                                                                                                                                                                                                                                                                                                                                                                |                          |
```

Comment:
typo in "where". Also, why the trailing "Harbor"? Same in line below.

Reply:
good catch thanks, I've just resolved it!